pom.xml

```diff
@@ -370,13 +392,13 @@
 
     <properties>
         <carbon.datasource.version>1.1.1-SNAPSHOT</carbon.datasource.version>
-        <carbon.feature.plugin.version>2.0.1</carbon.feature.plugin.version>
+        <carbon.feature.plugin.version>3.0.0</carbon.feature.plugin.version>
         <commons.io.version>2.4.0.wso2v1</commons.io.version>
         <org.apache.felix.version>1.0.0</org.apache.felix.version>
         <com.zaxxer.hikaricp.version>2.4.1</com.zaxxer.hikaricp.version>
         <java.version>1.8</java.version>
-        <carbon.kernel.version>5.1.0-alpha2</carbon.kernel.version>
-        <carbon.jndi.version>1.0.0-alpha</carbon.jndi.version>
+        <carbon.kernel.version>5.2.0-m3</carbon.kernel.version>
+        <carbon.jndi.version>1.0.2-SNAPSHOT</carbon.jndi.version>
```

Comment:
Please add the jndi release version after merging https://github.com/wso2/carbon-jndi/pull/33.